app/controllers/admin/organisations_controller.rb

```diff
@@ -70,4 +70,19 @@ def load_countries_for_dropdown
   def load_available_countries
     @available_countries = Country.with_organisations
   end
+
+  def updated_params
+    adapter_attributes = if params[:access_to_all]
+                           organisation_params[:adapter_attributes].merge(
+                             countries_with_access_ids: @available_countries.pluck(:id)
```

Comment:
This would be fine if we could assume all countries that are ever going to join EPIX are already signed up; in reality I don't think that would be the case, countries will be joining one by one. If this was a parameter of the adapter, we could use it for countries already in the system any countries joining in the future.